package.json

```diff
@@ -27,6 +27,9 @@
     "url": "https://github.com/Shopify/twine/issues"
   },
   "homepage": "https://github.com/Shopify/twine",
+  "dependencies": {
+    "jquery": "2.2.0"
```

Comment:
This is going to add a `jquery@2.2.0` dependency to anything consuming Twine.  Chances are that npm will resolve to the user's preferred version, but this still seems odd to me.

If we're not directly including it as part of Twine, maybe it'd be better as a `peerDependency`, with a `^2.2.0` to make it less specific?  

cc @lemonmade because I know you don't like peers :P